src/components/dropdownSearch/DropdownSearchInfiniteScrollOptions.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as InfiniteScroll from 'react-infinite-scroll-component';
+import * as _ from 'underscore';
+
+export interface DropdownSearchInfiniteScrollOptionsProps {
+    onMouseEnter: () => void;
+    options: JSX.Element[];
+    ulElementRefFunction: (menu: HTMLElement) => void;
+    infiniteScroll: InfiniteScroll.InfiniteScrollProps;
+}
+
+export class DropdownSearchInfiniteScrollOptions extends React.Component<DropdownSearchInfiniteScrollOptionsProps, {}> {
+    private id: string;

```

Comment:
why can't you have a normal id in the props instead of setting it in the props ?

Reply:
I only need it inside the component. It is used to specify the scrollable target. It has no other use. It is easier to define it once here than having to pass it every time we want a `DropdownSearchInfiniteScrollOptions`.